tests/sentry/snuba/metrics/test_query_builder.py

```diff
@@ -844,3 +845,32 @@ def test_translate_results_missing_slots(_1, _2, monkeypatch):
 def test_get_intervals():
     with pytest.raises(AssertionError):
         list(get_intervals(MOCK_NOW - timedelta(days=1), MOCK_NOW, -3600))
+
+
+def test_translate_meta_results():
+    meta = [
+        {"name": "p50(d:transactions/measurements.lcp@millisecond)", "type": "Array(Float64)"},
+        {"name": "tags[9223372036854776020]", "type": "UInt64"},
+        {"name": "project_id", "type": "UInt64"},
+    ]
+    assert translate_meta_results(meta) == [
+        {"name": "p50(transaction.measurements.lcp)", "type": "Array(Float64)"},
+        {"name": "transaction", "type": "UInt64"},
+        {"name": "project_id", "type": "UInt64"},
+    ]
+
+
+def test_translate_meta_results_with_duplicates():
+    meta = [
+        {"name": "p50(d:transactions/measurements.lcp@millisecond)", "type": "Array(Float64)"},
+        {"name": "p50(d:transactions/measurements.lcp@millisecond)", "type": "Array(Float64)"},
+        {"name": "tags[9223372036854776020]", "type": "UInt64"},
+        {"name": "tags[9223372036854776020]", "type": "UInt64"},
+        {"name": "project_id", "type": "UInt64"},
+        {"name": "project_id", "type": "UInt64"},
+    ]
+    assert translate_meta_results(meta) == [
+        {"name": "p50(transaction.measurements.lcp)", "type": "Array(Float64)"},
+        {"name": "transaction", "type": "UInt64"},
```

Comment:
@ahmedetefy i know we dicussed it, but i can't find correct answer in code. I need valid name of `String` type here.